src/fidesops/ops/service/email/email_dispatch_service.py

```diff
@@ -103,5 +103,5 @@ def _mailgun_dispatcher(
                 f"Email failed to send with status code {response.status_code}"
             )
     except Exception as e:
-        logger.error(e)
-        raise EmailDispatchException(format("Email failed to send: %s", str(e)))
+        logger.error("Email failed to send: %s", str(e))
+        raise EmailDispatchException(f"Email failed to send due to: {e}")
```

Comment:
I do think the %-style formatting you were using is better, there's a ticket to move to that at some point, but no need to change now since it's consistent with what we have now. 

https://github.com/ethyca/fidesops/issues/837